doc/classes/@GDScript.xml

```diff
@@ -326,7 +326,8 @@
 			<argument index="0" name="s" type="float">
 			</argument>
 			<description>
-				Raises the Euler's constant [b]e[/b] to the power of [code]s[/code] and returns it. [b]e[/b] has an approximate value of 2.71828.
+				The natural exponential function. It raises the mathematical constant [b]e[/b] to the power of [code]s[/code] and returns it.
+				[b]e[/b] has an approximate value of 2.71828.
```

Comment:
This will create another paragraph. (One line is one paragraph)

If this is intended, then I'm fine with it. Just letting you know why it was on the same line before. :)